### PR TITLE
Added more proper JSON field representation for wtforms.

### PR DESCRIPTION
**Pull request by fedExpress**
My idea is to replace current JSON field representation for wtforms-alchemy I did earlier.

### CODE REVIEW

File: wtforms_components/fields/json_field.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+from wtforms import fields, widgets
+
+
+class JSONField(fields.StringField):
+    """
+    A text field which stores a `json`.
+    """
+    widget = widgets.TextArea()
+
+    def __init__(self, label=None, validators=None, **kwargs):
```
Comment (tvuotila) [inline wtforms_components/fields/json_field.py]: You don't have to define the `__init__` method. You can remove it completely.

Comment (fedExpress) [reply wtforms_components/fields/json_field.py]: Removed. Thanks.

**Comment by fedExpress** [comment]
@tvuotila @kvesteri can you, please, merge these changes?

**Comment by fedExpress** [comment]
@tvuotila , @kvesteri, could you please, review these changes and merge if it is possible? That's definitely would be great.

**Comment by tvuotila** [comment]
LGTM

I will wait for @kvesteri decision on if this is in the scope of the package.

**Pull request merged**